app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
provide type annotations here

Reply:
I don't understand what type of annotations to provide here?